eodag/utils/stac_reader.py

```diff
@@ -50,7 +50,7 @@ def read_local_json(url, as_json):
                     return json.load(f)
                 else:
                     return f.read()
-        except FileNotFoundError:
+        except (FileNotFoundError, OSError):
```

Comment:
```suggestion
        except OSError:
```
`FileNotFoundError` is a subclass of `OSError`
See https://docs.python.org/3/library/exceptions.html#FileNotFoundError